apps/api/internal/api/v1/rewards/service.go

```diff
@@ -18,7 +19,7 @@ func handleGet(channelId string) ([]helix.ChannelCustomReward, error) {
 	config := do.MustInvoke[cfg.Config](di.Provider)
 	tokensGrpc := do.MustInvoke[tokens.TokensClient](di.Provider)
 
-	twitchClient, err := twitch.NewUserClient(channelId, config, tokensGrpc)
+	twitchClient, err := twitch.NewUserClient(context.Background(), channelId, config, tokensGrpc)
```

Comment:
There is `NewUserCLientWithContext`

Reply:
propagated context, changed to NewUserClientWithContext